requirements.txt

```diff
@@ -22,7 +22,7 @@ tabulate
 evaluate
 rouge_score
 pyyaml==6.0.1
-faiss-gpu
+faiss-gpu; python_version < '3.11'
```

Comment:
why python has to be less than 3.11, this is almost 6 months old stable python version.

Reply:
I think your change needs to be updated here in pip packages 
https://pypi.org/project/llama-recipes/#history which has been updated on 23rd july. I tested fine tuning notebook and currently its broken.